.ci/env/apt.sh

```diff
@@ -54,35 +54,80 @@ function install_dev-base-conda {
     conda env create -f .ci/env/environment.yml
 }
 
-function install_arm-cross-compilers {
-    sudo apt-get install -y gcc-aarch64-linux-gnu g++-aarch64-linux-gnu gfortran-aarch64-linux-gnu
+function install_gnu-cross-compilers {
+    sudo apt-get install -y "gcc-$1-linux-gnu" "g++-$1-linux-gnu" "gfortran-$1-linux-gnu"
 }
 
-function install_qemu_emulation {
+function install_qemu_emulation_apt {
     sudo apt-get install -y qemu-user-static
 }
 
+function install_qemu_emulation_deb {
+    sudo apt-get install -y binfmt-support
+    qemu_deb=qemu-user-static_8.2.2+ds-2+b1_amd64.deb
+    wget http://ftp.de.debian.org/debian/pool/main/q/qemu/${qemu_deb}
```

Comment:
Is there an https link instead? I'm not sure how the signature of the package is checked otherwise.

Reply:
Good question. I have checked just now with wget, and see the following error message
```
Connecting to ftp.de.debian.org (ftp.de.debian.org)|141.76.2.4|:443... connected.
ERROR: no certificate subject alternative name matches
        requested host name ‘ftp.de.debian.org’.
To connect to ftp.de.debian.org insecurely, use `--no-check-certificate'.
```

So it looks like there is no signature verification for the package